Jenkinsfile

```diff
@@ -115,17 +114,16 @@ pipeline {
             stage('Legacy build') {
               steps {
                 runningOn(env.NODE_NAME)
+                sh 'git clone git@github.com:xmos/xmos_cmake_toolchain'
```

Comment:
Low risk, but could be worth using a tag on the clone here and line 192.

Reply:
I'll clone out v1.0.0